### ISSUE TITLE
Not working with latest mongodb version

### ISSUE THREAD

**Issue by MaryCoding** [zumba/mongounit#11]
Hi, 
I am currently trying to use `zumba/mongounit` with my current set up but I am getting the below error. I have the latest version of `mongodb/mongodb` installed(which is  `1.2`) but based on the error shwon below it looks like  `zumba/mongounit` only supports an earlier version of mongodb. Could it be possible to add support for mongodb 1.2. Thanks!

```
  Problem 1
    - zumba/mongounit 3.0.0 requires mongodb/mongodb ~1.1.1 -> satisfiable by mongodb/mongodb[v1.1.x-dev].
    - zumba/mongounit 3.0.1 requires mongodb/mongodb ~1.1.1 -> satisfiable by mongodb/mongodb[v1.1.x-dev].
    - zumba/mongounit 3.0.2 requires mongodb/mongodb ~1.1.1 -> satisfiable by mongodb/mongodb[v1.1.x-dev].
    - Conclusion: don't install mongodb/mongodb v1.1.x-dev
    - Installation request for zumba/mongounit ^3.0 -> satisfiable by zumba/mongounit[3.0.0, 3.0.1, 3.0.2].
```

**Comment by cjsaylor** [comment]
Hi @MaryCoding, @rossaffandy's PR should resolve the issue, however it is pending some review feedback. As soon as that is resolved and the PR is merged, mongodb 1.2 should begin to work as a dependency.

**Comment by cjsaylor** [comment]
@MaryCoding This fix for this issue is available in 3.0.3.
